storage/country_info_getter.cpp

```diff
@@ -59,6 +59,20 @@ TCountryId CountryInfoGetter::GetRegionCountryId(m2::PointD const & pt) const
   return id != kInvalidId ? m_countries[id].m_countryId : kInvalidCountryId;
 }
 
+vector<TCountryId> CountryInfoGetter::GetRegionsCountryIdByRect(m2::RectD const & rect) const
+{
+  size_t constexpr kAverageSize = 10;
```

Comment:
If you know average size (could you please explain how you get this constant?) why not to use buffer_vector instead?